feat(srv): add troubleshooting pages MTA-5446 (#5280)
scaleway/docs-content: pages/serverless-containers/troubleshooting/cannot-access-container.mdx

---

> meta:

Reviewer: Meta needs updating for new guidelines

  title: I cannot access my container
  description: Troubleshoot issues encountered while accessing or connecting to your Scaleway Serverless Containers.
content:
  h1: I cannot access my container
  paragraph: Troubleshoot issues encountered while accessing or connecting to your Scaleway Serverless Containers.
tags: serverless containers troubleshooting issue error state unreachable connection log in access
dates:
  validation: 2025-07-15
  posted: 2025-07-15
categories:
  - serverless
---

## Problem

I am experiencing issues while accessing my Serverless Containers.

## Possible causes
	

> - The container returns A DNS error or issue

Reviewer: ```suggestion
- The container returns a DNS error or issue
```

- The container response time is abnormally long
- The container returns a `HTTP 5xx` error
- The container returns a `Could not resolve host` error
- The container is in an error state

## Solutions

- If you encounter a DNS issue, refer to the [dedicated troubleshooting page](/serverless-containers/troubleshooting/known-dns-containers-errors/).

- For long response time, check:

>     - if the first instance of the container is currently starting, a [cold start](/serverless-containers/concepts/#cold-start) can appear

Reviewer: ```suggestion
    - if the first instance of the container is currently starting. This can entail a [cold start](/serverless-containers/concepts/#cold-start)
```

    - if not, [check the metrics in Cockpit](/serverless-containers/how-to/monitor-container/) (especially CPU) and see if there is a high usage explaining the slow response

> - `HTTP 5XX` errors can come from the application itself. Check if you can reproduce locally with a dummy container, or if your application is not misbehaving. 

Reviewer: ```suggestion
- `HTTP 5XX` errors can come from the application itself. Check if you can reproduce locally with a dummy container, or if your application is misbehaving. 
```
